service/local-dev/README.md

```diff
@@ -1,15 +1,10 @@
-This directory contains scripts for running continuous development builds. This 
+This directory contains scripts for running continuous development builds. This
 is not necessary for deployment, but it can be helpful for developing faster.
+
 The provided setup script clones the terra-helm and terra-helmfile git repos,
 and templates in the desired Terra environment/k8s namespace to target.
 If you need to pull changes to either terra-helm or terra-helmfile, rerun this script.
 
-To use this, first ensure [Helm](https://helm.sh/docs/intro/install/#from-homebrew-macos) and [Skaffold](https://skaffold.dev/) are installed on your local machine. 
```

Comment:
I think we now need `skaffold`, `helm` and `helmfile` installed on the local machine for this to work.
Maybe add that to the documentation?

Reply:
The `terra-helmfile/bin/render` is a wrapper around a Docker container, so you should only need Skaffold and Docker

Reply:
Nice